packages/openneuro-server/src/graphql/resolvers/dataset.js

```diff
@@ -23,8 +23,8 @@ import { getDatasetWorker } from '../../libs/datalad-service.js'
 import { getDraftHead } from '../../datalad/dataset.js'
 import { getFileName } from '../../datalad/files.js'
 
-export const dataset = (obj, { id }, { user, userInfo }) => {
-  return checkDatasetRead(id, user, userInfo).then(() => {
+export const dataset = async (obj, { id }, { user, userInfo }) => {
+  return await checkDatasetRead(id, user, userInfo).then(() => {
```

Comment:
This is also an extra async/await. It could be refactored to use await instead of a promise chain within the await.